evm/utils/datatypes.py

```diff
@@ -105,7 +109,7 @@ def configure(cls,
                     )
                 )
 
-            configured_sub_cls = sub_cls.configure(**sub_overrides)
+            configured_sub_cls = sub_cls.configure(**sub_overrides)  # type: ignore
```

Comment:
Why do we have to tell mypy to ignore this line? If there's no way around it, can you add a comment explaining why?

Reply:
`mypy` doesn't know that `sub_cls` is of (sub)type `Configurable`.

We could also initialize it to `None` before we first introduce it and add a comment type hint to set it to `Configureable`.

```
        for key, sub_overrides in sub_overrides_by_prop.items():
            sub_cls = None # type: Configurable
```

Do you have any other idea how to best resolve this?